doc/en/components/grids/grid/groupby.md

```diff
@@ -43,6 +43,28 @@ constructor() {
 ```
 <!-- end: WebComponents -->
 
+<!-- React -->
+```typescript
+const expressions = [
+    { fieldName: 'ProductName', dir: SortingDirection.Desc },
+    { fieldName: 'Released', dir: SortingDirection.Desc }
+];
+
+function App() {
+    const grid1Ref = useRef();
+    return (
+    <>
+        <IgrGrid
+            autoGenerate="true"
+            groupingExpressions={expressions}
```

Comment:
I don't get this underlined because of a typing error in 18.2.2-beta.0 but I can't test if it works because declarative grouping throws in that version heavily. I'll test again when we have a newer beta. For now I'd say we should leave it as this is only ever going to be a product bug. The snippet shows how it should work for sure.